templates/devtest/tmplerr-sub.tmpl

```diff
@@ -0,0 +1,3 @@
+sub template triggers an executing error
```

Comment:
`templates/test`? They aren't really used anywhere that I see but I guess we can start test-only templates.

Reply:
It's only enabled in dev mode, it helps development and testing, so what's why it's called `devtest`, and this word is unique in the codebase, people can easily grep it out.


![image](https://user-images.githubusercontent.com/2114189/231858301-a13900f4-7ca5-4fd2-a4a4-57fa936c0cbe.png)

Reply:
I see, it's already in use, so lets keep it.